syntax = "proto3";

package example.complex;

option go_package = "github.com/tbh26/harbor/modern_api/protobuf/go_intro/proto";

message Dum {
  int32 id = 1;
  string name = 2;
}

message Complex {
  Dum one_dummy = 2;
  repeated Dum multiple_dummies = 3;
}
